wurst/util/DialogBox.wurst

```diff
@@ -0,0 +1,84 @@
+package DialogBox
+import HashMap
+import Dialog
+import LinkedList
+
+public interface Callback
```

Comment:
The name is too generic, especially so since it's a public interface. Since it looks like it is only used internally, you could argue that it can simply not be public, however, considering that it is used as an argument (which can also be created and saved outside this system as variables), it should remain public, but change name, to something like DialogBoxButtonClosure. It looks ugly and overly verbose, but closure names are typically long and descriptive because they both kind of need to be in public space, and need to be sound specific enough in order not to clash with others. Luckily, users rarely need to manually write them down, so they're mostly bound to systems they are part of.